### PR TITLE
UNITY-5970: Add multiple lineitems to the cart

### PR DESCRIPTION
**Pull request by mhoehn**

### CODE REVIEW

File: _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+      "currency": "EUR"
+    },
+    "productLineItems": [
+      {
+        "lineItemId": "5858EDE0-6E6D-DBC6-A97B-D5809AB37DD9",
+        "sku": "Acc-001",
+        "name": "Connector 6.3 mm",
+        "productId": "54E72E1F-746A-8B6E-8E41-0A0C05E64669",
+        "quantity": {
+          "amount": 2,
```
Comment (cgebken) [inline _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json]: Shouldn't this be a 1?

File: _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+        "energyLabelsMaxEfficiencyString": "A+++",
+        "energyLabelSourceFile": "https://pm.epages.com/WebRoot/Store/Shops/apidocu/Products/be_40401/label-1-9dee85f24e1.png"
+      },
+      {
+        "lineItemId": "44EF2GHF-746A-8B6E-8E41-0A0C05E62787",
+        "sku": "Ccd-001",
+        "name": "Connector 12 mm",
+        "productId": "66E72E1F-746A-8B6E-8E41-0A0C05E87456",
+        "quantity": {
+          "amount": 12,
```
Comment (cgebken) [inline _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json]: Shouldn't this be a 6?

File: _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json (hunk 1 above)
Comment (mhoehn) [reply _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json]: yes

File: _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json (hunk 2 above)
Comment (mhoehn) [reply _raml/apps/carts/examples/post_carts_cartid_multilineitem_response.json]: yes

**Pull request merged**